core/templates/pages/lightweight-oppia-root/app.module.ts

```diff
@@ -32,7 +32,43 @@ import { LightweightOppiaRootComponent } from './lightweight-oppia-root.componen
 import { platformFeatureInitFactory, PlatformFeatureService } from 'services/platform-feature.service';
 import { RequestInterceptor } from 'services/request-interceptor.service';
 import { CookieModule } from 'ngx-cookie';
+import { ToastrModule } from 'ngx-toastr';
 import { AppErrorHandlerProvider } from 'pages/oppia-root/app-error-handler';
+// This throws "TS2307". We need to
+// suppress this error because hammer come from hammerjs
+// dependency. We can't import it directly.
+// @ts-ignore
+import * as hammer from 'hammerjs';
+
+// Config for ToastrModule (helps in flashing messages and alerts).
+export const toastrConfig = {
+  allowHtml: false,
+  iconClasses: {
+    error: 'toast-error',
+    info: 'toast-info',
+    success: 'toast-success',
+    warning: 'toast-warning'
+  },
+  positionClass: 'toast-bottom-right',
+  messageClass: 'toast-message e2e-test-toast-message',
+  progressBar: false,
+  tapToDismiss: true,
+  titleClass: 'toast-title'
+};
+
+export class MyHammerConfig extends HammerGestureConfig {
```

Comment:
Can this class be renamed to `HammerConfig`?

Reply:
I tried following the codebase naming convention. `oppia-root` uses `MyHammerConfig`. But I have now made the change.